lib/components/taskbar/taskbar_element.dart

```diff
@@ -48,9 +48,7 @@ class _TaskbarElementState extends State<TaskbarElement> {
     return SizedBox.fromSize(
       size: widget.size ?? const Size(48, 48),
       child: GestureDetector(
-        onTap: () => widget.overlayID != null
-            ? shell.toggleOverlay(widget.overlayID!)
-            : {},
+        onTap: () => widget.overlayID != null ? shell.toggleOverlay(widget.overlayID!) : {},
```

Comment:
widget.overlayID != null ? () => shell.toggleOverlay(widget.overlayID!) : null

as it is right now it won't work properly